sirepo/package_data/static/js/raydata.js

```diff
@@ -540,6 +540,30 @@ SIREPO.app.directive('scansTable', function() {
                 }
             }
 
+            function decodeSortColumns(encoded) {
+                let columns = new Map();
+                encoded.split(';').forEach((e) => {
```

Comment:
What about storing the encoded value using a 2d array? That way we don't have to with commas and semicolons. And we get things like true/false for free.